drivers/dai/intel/dmic/dmic.h

```diff
@@ -8,21 +8,12 @@
 #define __INTEL_DAI_DRIVER_DMIC_H__
 
 #include <zephyr/sys/util_macro.h>
-
-/* bit operations macros */
-#define MASK(b_hi, b_lo)	\
-	(((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL) << (b_lo))
+#include <zephyr/sys/util.h>
 
 #define SET_BIT(b, x)		(((x) & 1) << (b))
 
-#define SET_BITS(b_hi, b_lo, x)	\
-	(((x) & ((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL)) << (b_lo))
-
-#define GET_BIT(b, x) \
-	(((x) & (1ULL << (b))) >> (b))
-
 #define GET_BITS(b_hi, b_lo, x) \
```

Comment:
It would be useful to move it to the `util_macro.h` and make that macro public since some drivers implement it in their own way.